### PR TITLE
Create Leal2-6465.md

### PR DESCRIPTION
**Pull request by Lealdos**
# Create Jose L Delgado Avila profile

### CODE REVIEW

**Review by ntoniocp — APPROVED** [review]

**Review by ntoniocp — APPROVED** [review]
Test

File: github-profiles/Leal2-6465.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# My github profile  https://github.com/Lealdos
```
Comment (glrodasz) [inline github-profiles/Leal2-6465.md:1]: This file shouldn't exist, because it has been move into the folder instead.

**Review by glrodasz — CHANGES_REQUESTED** [review]
Please fix the comments.

**Comment by glrodasz** [comment]
I couldn't find the user on Discord it should be `leal2#6465`?

File: github-profiles/Leal2-6465.md (hunk 1 above)
Comment (Lealdos) [reply github-profiles/Leal2-6465.md]: Hello Guillermo, I fixed the error and deleted the leftover file


image

**Comment by Lealdos** [comment]
Hello Guillermo, ty for your comments I already fixed them, and in my last commit I added the CSS file with styles for the index.html

Yes, my discord profile is **Leal2#6465**

image

**Comment by glrodasz** [comment]
@Lealdos Excellent, there are just two little details left before merging: 

1. You delete the root README.md file, that one we should keep it.
2. rename your own `readme.md` to `README.md` so we keep the convention.

**Comment by Lealdos** [comment]
Hello, @glrodasz README.md created in the root and name file in **lealdos-6465 modified** to UPPERCASES

**Review by Josofico — APPROVED** [review]

**Review by Lealdos — COMMENTED** [review]
I already did the changes that you asked for

**Pull request merged**